src/workload/client.rs

```diff
@@ -0,0 +1,148 @@
+use std::collections::HashMap;
```

Comment:
Is this suppose to be a redis client? If so we should name it as such.

Reply:
Do you think the 'protocol' should also be referred to as redis? Or should there be an implicit agreement that when you specify 'resp' protocol, that a redis client is used?

Reply:
I think either `redis` or `resp` would work, no opinion on that. Basically I'm thinking what happens when we add another workload/client to this directory.

Reply:
It's been renamed to `redis` in the latest version.